bin/nbb

```diff
@@ -344,6 +339,24 @@ find_centos_7_ami_ids() {
   done
 }
 
+shutdown_instances() {
+  aws ec2 describe-instances --filters "Name=instance-state-name,Values=running" | jq -c -r '.Reservations[] .Instances[] .InstanceId' > instance_list
+  echo "You are about to terminate the following instances:"
+  cat instance_list
+  read -r -p "Are you sure? [y/N] " response
```

Comment:
Suggestion: allow the user to pass in an option like `--yes-im-sure`; if _and only if_ that option is present (e.g. it's **not** the default), don't bother asking. Allows for semi-scriptable interface and/or some form of automation. Might not be advisable or obvious how it's useful but I always come at these things from the perspective of "_everything_ should be able to run non-interactively, no matter what, even if it's dangerous - as long as you force the user to opt-in to the dangerous stuff first".